### PR TITLE
Implement services

### PR DESCRIPTION
**Pull request by lnogueir**
Implement LASN services

### CODE REVIEW

File: services/lasn/lasn.go [hunk 1]
```diff
@@ -3,27 +3,118 @@ package lasn
...
+	if err != nil {
+		log.Printf("Error while collecting results: [%v]", err)
+		return nil, err
+	}
+	log.Printf("Fetch return %v records", len(userSummaries))
+	return &protos.FetchResponse{UserSummaries: userSummaries}, nil
+}
+
+func (s *Service) Disconnect(ctx context.Context, request *protos.DisconnectRequest) (*protos.DisconnectResponse, error) {
+	// TODO: write query to remove all edges from node identified by request.UserId and link dangling components.
```
Comment (lnogueir) [inline services/lasn/lasn.go:65]: @SergioSanchez12 can you work on this? I added a question on Stack Overflow with details to the problem: https://stackoverflow.com/questions/73008423/link-unconnected-components-after-deleting-bridge-node

File: managers/neo4j.go [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+
+func (m *Neo4jManager) CreateOrUpdateNode(userSummary *protos.UserSummary) error {
+	_, err := m.executeQuery(fmt.Sprintf(`
+			MERGE ({
+				userid: "%v",
+				deviceid: "%v",
+				name: "%v",
+				birthdate: %v,
+				thumbnail: "%v"
+			})
```
Comment (lnogueir) [inline managers/neo4j.go:40]: @SergioSanchez12 Can you work on this query to make sure there is a constraint that `userid` is unique and that if `userid` exists this actually updates that existing node and not create a new one.

File: services/lasn/lasn.go (hunk 1 above)
Comment (lnogueir) [reply services/lasn/lasn.go:65]: Someone posted a solution that works there ^^

**Review by SergioSanchez12 — APPROVED** [review]
lgtm

File: managers/neo4j.go (hunk 2 above)
Comment (lnogueir) [reply managers/neo4j.go:40]: Added a TODO.

**Pull request merged**